pkg/blob/controllerserver.go

```diff
@@ -691,3 +776,28 @@ func parseDays(dayStr string) (int32, error) {
 
 	return int32(days), nil
 }
+
+// generateSASToken generate a sas token for storage account
+func generateSASToken(accountName, accountKey, storageEndpointSuffix string) (string, error) {
+	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
+	if err != nil {
+		return "", status.Errorf(codes.Internal, fmt.Sprintf("failed to generate sas token in creating new shared key credential, accountName: %s, accountKey: %s, err: %s", accountName, accountKey, err.Error()))
+	}
+	serviceClient, err := service.NewClientWithSharedKeyCredential(fmt.Sprintf("https://%s.blob.%s/", accountName, storageEndpointSuffix), credential, nil)
+	if err != nil {
+		return "", status.Errorf(codes.Internal, fmt.Sprintf("failed to generate sas token in creating new client with shared key credential, accountName: %s, accountKey: %s, err: %s", accountName, accountKey, err.Error()))
+	}
+	sasURL, err := serviceClient.GetSASURL(
+		sas.AccountResourceTypes{Object: true, Service: true, Container: true},
+		sas.AccountPermissions{Read: true, List: true, Write: true},
+		sas.AccountServices{Blob: true}, time.Now(), time.Now().Add(1*time.Hour))
```

Comment:
`1*time.Hour` is still too long time, pls do some tests and add feature flag in the driver to control the time

Reply:
add a flag name sasExpiryTime for control and default is 90 minutes